ansible/roles/postgres-migration/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# tasks file for postgres-migration
+- name: Starting flyway for migration
+  become: yes
+  shell: "docker run --network=host --rm -v {{ role_path }}/files:/flyway/sql -v {{ role_path }}/conf:/flyway/conf flyway/flyway -url=jdbc:postgresql://{{postgresql_migration_endpoint}}/{{postgresql_migation_db}} -user={{postgresql_migration_user}} -password={{postgresql_migration_password}} migrate"
```

Comment:
We already have Postgres connection variables defined in the repo. Please reuse them rather than redefining new variables.
Also rather than using `shell` please use the docker module - https://docs.ansible.com/ansible/latest/collections/community/docker/docker_container_module.html

Reply:
makes it complicated